toolkit/tools/specreader/specreaderutils/specreaderutil.go

```diff
@@ -0,0 +1,706 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
This module should live under `/toolkit/tools/pkg/specreaderutils`

Reply:
Thoughts on renaming `pkg` to `utils`, so that we don't confuse it with the package-build related tooling?

Reply:
`pkg` is the idiomatic name used in Go.